app/components/CategoryOverviewView.js

```diff
@@ -4,9 +4,9 @@ import { View } from 'react-native';
 
 import CategoryHeaderAndroid from './CategoryHeader.android';
 
-const CategoryOverviewView = ({ category, platform }) => (
+const CategoryOverviewView = ({ category, platform, openDrawer }) => (
   <View>
-    {platform === 'android' ? <CategoryHeaderAndroid category={category} /> : null}
+    {platform === 'android' ? <CategoryHeaderAndroid category={category} openDrawer={openDrawer} /> : null}
```

Comment:
I think `openDrawer` should be renamed to reflect component's POV, `onToolbarIconClick` or smth like that.